.gitignore

```diff
@@ -30,4 +30,9 @@ common/dist/scripts/*
 common/dist/sprites/habitrpg-shared.css
 .elasticbeanstalk/config.yml
 
+
 test/spec/mocks/translations.js
+
+#vim
+.tern-project
```

Comment:
I wouldn't worry try rebasing unless you know what you're doing. This should be sufficient:

```
git revert d85410607229a5b7541112a1bd32abaab5d9d041
git push
```

We can squash the commits when we merge it in.